test-framework/test-suites/integration/tests/report/test_report_host.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+
+@pytest.mark.usefixtures("revert_database")
+class TestReportHost:
+    def test_report_host_with_default_network_behaviour():
+        result = host.run(f'stack add host backend-0-0')
+        assert result.rc == 0
+        result = host.run(f'stack add host interface backend-0-0 interface=eth0 ip=10.10.11.11 network=private')
+        assert result.rc == 0
```

Comment:
I would put a newline after each assert, to visually group it with the run command above it.